libpermian/webui/static/style.css

```diff
@@ -113,10 +113,12 @@ h1 {
 
 .crc_logs ul {
     max-height: 2.2em;
-    overflow-y: hidden;
+    max-width: 150px;
+    overflow: hidden;
     text-align: left;
     list-style-position: inside;
     padding: 0px;
+    white-space: nowrap;
```

Comment:
For some reason, I don't see this option applied when testing it. investigating.

Reply:
Ok, that was issue on my end with cached CSS file.